README.md

```diff
@@ -1,5 +1,5 @@
-1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task_clone_table_column_DOM/)
+1. Replace `robsza1122` with your Github username in the link
+    - [DEMO LINK](https://robsza1122.github.io/js_task_clone_table_column_DOM/)
```

Comment:
Add this link to the PR description
<img width="934" alt="image" src="https://github.com/mate-academy/js_task_clone_table_column_DOM/assets/64750958/7b47a03b-8e2f-4558-8570-28858bf7786f">